### PR TITLE
Update Chinese l18n (1.3.4 dev2)

### PR DESCRIPTION
**Pull request by hedw1gP**

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -24,9 +24,11 @@
 
 ## 视频
 
-这是一个关于 Aperture 的播放列表，内含更新和教程。（由Noble团队制作的[中文教程](http://www.bilibili.com/video/av15303499/)不代表 McHorse 观点，内容可能过时）
+这是一个关于 Aperture 的播放列表，内含更新和教程。
 
-<a href="https://youtu.be/_vfZYtiRfRs?list=PL6UPd2Tj65nFLGMBqKaeKOPNp2HOO86Uw"><img src="https://img.youtube.com/vi/_vfZYtiRfRs/0.jpg"></a> 
+<a href="https://youtu.be/5Tnp4z6GWME?list=PL6UPd2Tj65nFLGMBqKaeKOPNp2HOO86Uw"><img src="https://img.youtube.com/vi/5Tnp4z6GWME/0.jpg"></a> 
+
+也可前往 [McHorse's Mods 哔哩哔哩中文频道](https://space.bilibili.com/472615413) 观看。
```
Comment (hedw1gP) [inline README_CN.md:31]: Add a link to Bilibili channel

**Comment by mchorse** [comment]
Looks good, however, can you also edit [this](https://github.com/mchorse/aperture/compare/1.3.3...master#diff-6ebf47e5765faf9f5ecf471eef6e26b0R30)?

**Comment by hedw1gP** [comment]
Thank you so much! @Chunk7w

**Comment by mchorse** [comment]
Awesome, guys, thank you very much! 👏 😁

**Pull request merged**